static/js/vue-cdr-access/tests/unit/streamingPlayer.spec.js

```diff
@@ -27,18 +46,33 @@ describe('streamingPlayer.vue', () => {
     });
 
     // Default record url is set in jest.config.js, which is where the refUrl comes from here
-    it('has a source link for the streaming player', () => {
-        expect(wrapper.find('#streaming-player').attributes('src')).toEqual(`${streamingUrl}&refUrl=https://localhost/record/73bc003c-9603-4cd9-8a65-93a22520ef6a`);
+    it('has a source link for the streaming player', async () => {
```

Comment:
For all three of the tests updated in this PR, I'd suggest splitting them into separate tests for the file versus work page, it should help with readability and they seem like separate test cases.